minifi-toolkit/minifi-toolkit-configuration/src/test/resources/ProcessGroupsAndRemoteProcessGroups.yml

```diff
@@ -173,13 +173,14 @@ Process Groups:
       max work queue data size: 1 GB
       flowfile expiration: 0 sec
       queue prioritizer class: ''
-    Remote Processing Groups:
+    Remote Process Groups:
```

Comment:
Definitely good to update expected output for test ymls, where possible though it would be nice to add a -v1.yml so we validate the upconvert works as expected.  (ex: DecompressionCircularFlow-v1.yml vs DecompressionCircularFlow.yml)

It should be automagically picked up by ConfigMainTest.testV1YmlIfPresent() if named via the convention.